native/src/main/java/io/ballerina/stdlib/http/api/HttpUtil.java

```diff
@@ -1352,6 +1352,10 @@ public static void populatePoolingConfig(BMap poolRecord, PoolConfiguration pool
                 maxActiveStreamsPerConnection == -1 ? Integer.MAX_VALUE : validateConfig(
                         maxActiveStreamsPerConnection,
                         HttpConstants.CONNECTION_POOLING_MAX_ACTIVE_STREAMS_PER_CONNECTION.getValue()));
+
+        long minEvictableIdleTime =
+                poolRecord.getIntValue(HttpConstants.CONNECTION_POOLING_EVICTABLE_IDLE_TIME);
+        poolConfiguration.setMinEvictableIdleTime(minEvictableIdleTime);
```

Comment:
Isn't this function sets the minimum time as milliseconds?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-http/pull/1841/commits/1b88b37adec5c89f232aa6b229cbfaa07613f4e1